src/System.IO.FileSystem.DriveInfo/src/System/IO/DriveInfo.Unix.cs

```diff
@@ -52,8 +69,9 @@ public string DriveFormat
             [SecuritySafeCritical]
             get
             {
-                Interop.libc.statfs data = Interop.libc.GetStatFsForDriveName(Name);
-                return Interop.libc.GetMountPointFsType(data);
+                string format = string.Empty;
+                CheckStatfsResultAndThrowIfNecessary(() => Interop.Sys.GetFormatInfoForMountPoint(Name, out format));
```

Comment:
It'd be better for CheckStatfsResultandThrowIfNecessary to take an int rather than a delegate, such that this property is:
```C#
string format;
CheckStatfsResultAndThrowIfNecessary(Interop.Sys.GetFormatInfoForMountPoint(Name, out format));
return format;
```
While these properties are unlikely to be performance bottlenecks, it's trivial here to avoid the unnecessary delegate and closure allocations, plus delegate invocation, and it's fewer characters on top of that ;)